CruzHacks/cruzhacks-2024-website
# Hacker Support

# Scope

- Add 2023 Live Site pages and support information

# Implementation

- Extract Navbar/footer from homepage/team to general static layout
- Create support pages
- Link support pages on portal
- Update Navbar to link to support

## TODO:
- [x] Add support sublinks to Navbar
- [x] Update Maps 2024
- [ ] Update links (need hacker pack)
- [ ] Update workshop schedule
- [x] Slack -> Discord

# How to Test

- Click support link on navbar on homepage
- Check links on Portal sidebar
